libyul/optimiser/DataFlowAnalyzer.cpp

```diff
@@ -253,6 +253,17 @@ void DataFlowAnalyzer::handleAssignment(set<YulString> const& _variables, Expres
 		// assignment to slot contents denoted by "name"
 		m_memory.eraseValue(name);
 	}
+
+	if (_value)
+	{
+		// This might erase additional knowledge about the slot.
+		// On the other hand, if we knew the value in the slot
+		// already, then the sload() / mload() would have been replaced by a variable anyway.
```

Comment:
```suggestion
		// This might erase additional knowledge about the slot.
		// On the other hand, if we know the value in the slot
		// already, then the sload() / mload() would have been replaced by a variable anyway.
```